README.md

```diff
@@ -232,6 +233,16 @@ cp configs/mesh-mbedignore ./.mbedignore
 
 If you have issues with the `X-NUCLEO-IDW04A1` board, please double-check that macro `IDW04A1_WIFI_HW_BUG_WA` has been added to the `macros` section of the `mbed_app.json` file.
 
+#### Compile configuration for REALTEK_RTL8195AM (aka Realtek Ameba) board
+
+Use the supplied `configs/wifi_rtw_v4.json` file as the basis.
+
+``` bash
+cp configs/wifi_rtw_v4.json mbed_app.json
+<use your favourite editor to modify mbed_app.json for WiFi SSUD/Password>
+mbed compile -m REALTEK_RTL8195AM -t <your_toolchain>
```

Comment:
`-t <TOOLCHAIN>`

to be inline with mbed-cli documentation https://github.com/armmbed/mbed-cli

Reply:
Done!